catalog/spatial/geocoding/spatial-geocoding-plugin/pom.xml

```diff
@@ -65,7 +72,7 @@
                             ddf.catalog.util.impl
                         </Private-Package>
                         <Embed-Dependency>
-                            commons-lang3
+                            commons-lang3, catalog-core-api-impl, platform-util
```

Comment:
These were some odd dependencies that were breaking the build. Travis knows more on why they are needed.